packages/contact-exchange/src/services/OOBTestFixtures.ts

```diff
@@ -0,0 +1,57 @@
+// OOBTestFixtures.ts
+import { OutOfBandInvitation } from '../services/DIDCommOOBInvitation';
+import { Contact } from '../services/Wallet';
+
+// OutOfBandInvitation fixtures
+export const validOutOfBandInvitation: OutOfBandInvitation = {
+  '@id': 'invitation-id',
+  '@type': 'https://didcomm.org/out-of-band/2.0/invitation',
+  services: [
+    {
+      id: 'did:example:123456789abcdefghi',
+      type: 'did-communication',
+      serviceEndpoint: 'http://example.com/endpoint',
+      recipientKeys: ['did:example:123456789abcdefghi#key-1'],
+    },
+  ],
+  '@cid': '',
+};
+
+export const invalidOutOfBandInvitation: OutOfBandInvitation = {
+  '@id': 'invitation-id',
+  '@type': 'https://didcomm.org/out-of-band/2.0/invitation',
+  services: [
+    {
+      id: 'did:example:123456789abcdefghi',
+      type: 'did-communication',
+      serviceEndpoint: 'http://example.com/endpoint',
+      recipientKeys: [],
+    },
+  ],
+  '@cid': '',
+};
+
+// Base64 Encoded URL fixtures
+export const validEncodedUrl =
+  'https://identity.foundation/didcomm-messaging/spec/#standard-message-encoding?eyJAY2lkIjoiSW52aXRhdGlvbi1JRCIsIkB0eXBlIjoiIHRlc3QifQ==';
```

Comment:
I am not sure about where is this format coming from... I think that the scope of this feature, should start with an OOB.

It has this format:
https://mediator.rootsid.cloud?_oob=eyJ0eXBlIjoiaHR0cHM6Ly9kaWRjb21tLm9yZy9vdXQtb2YtYmFuZC8yLjAvaW52aXRhdGlvbiIsImlkIjoiMDczODZhZTUtMjYzYi00ZjgwLWE2M2ItZmI5OTE1ODIzM2IyIiwiZnJvbSI6ImRpZDpwZWVyOjIuRXo2TFNtczU1NVloRnRobjFXVjhjaURCcFptODZoSzl0cDgzV29qSlVteFBHazFoWi5WejZNa21kQmpNeUI0VFM1VWJiUXc1NHN6bTh5dk1NZjFmdEdWMnNRVllBeGFlV2hFLlNleUpwWkNJNkltNWxkeTFwWkNJc0luUWlPaUprYlNJc0luTWlPaUpvZEhSd2N6b3ZMMjFsWkdsaGRHOXlMbkp2YjNSemFXUXVZMnh2ZFdRaUxDSmhJanBiSW1ScFpHTnZiVzB2ZGpJaVhYMCIsImJvZHkiOnsiZ29hbF9jb2RlIjoicmVxdWVzdC1tZWRpYXRlIiwiZ29hbCI6IlJlcXVlc3RNZWRpYXRlIiwibGFiZWwiOiJNZWRpYXRvciIsImFjY2VwdCI6WyJkaWRjb21tL3YyIl19fQ

as seen on this mediator:
https://mediator.rootsid.cloud

or on the spec:
https://identity.foundation/didcomm-messaging/spec/#standard-message-encoding

Please review.

Reply:
And therefore also review the parser method.

Reply:
And overall flow.